src/test/java/org/sonar/scm/git/blame/BlobReaderIT.java

```diff
@@ -38,4 +46,17 @@ public void loadText_whenWorkDirectoryAndFileDoesntExist_thenThrowISE() {
 
     Assert.assertThrows(IllegalStateException.class, () -> reader.loadText(objectReader, fc));
   }
+
+  @Test
+  public void loadText_whenWorkDirectoryHasDirectories_thenIgnoreDirs() throws IOException {
```

Comment:
minor but should it be : `loadText_whenWorkDirectoryHasDirectories_shouldIgnoreDirs` ?

Reply:
thanks for pointing that out, I've been using the wrong format in many places without realizing it

Reply:
fixed